Runes.podspec

```diff
@@ -0,0 +1,18 @@
+Pod::Spec.new do |spec|
+  spec.name = 'Runes'
+  spec.version = '1.1.1'
+  spec.summary = 'Indecipherable symbols that some people claim have actual meaning.'
```

Comment:
This is funny in the README, but can we change this to "Functional operators for Swift"?